jobs/kola-aws.Jenkinsfile

```diff
@@ -1,9 +1,10 @@
-def pipeutils, pipecfg, s3_bucket, official
+def pipeutils, pipecfg, s3_bucket, official, src_config_url
 node {
     checkout scm
     pipeutils = load("utils.groovy")
     pipecfg = pipeutils.load_pipecfg()
     s3_bucket = pipecfg.s3_bucket
+    src_config_url = pipecfg.source_config.url
```

Comment:
Cool with this, but I'd like to go the other direction actually where we directly reference e.g. `pipecfg.source_config.url` where needed instead of adding more and more aliases here. That was partly why I really wanted a short `pipecfg` name for the config.

Reply:
I'm cool with that too. It just takes this a step farther IIUC. Let's do that in a followup.